src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -205,6 +205,56 @@ def test_pqw_for_circular_equatorial_orbit():
     assert_allclose(w, expected_w)
 
 
+def test_frozen_orbit_argp():
```

Comment:
The only body that has J3 now is the Earth. Could we look for some other rocky body with a documented J3 value and add some tests that show that

1. One can create frozen orbits with attractors different than the Earth, and
2. If the given attractor does not have J3, the `AttributeError` is see from the outside?